BlueShift-iOS-SDK/InApps/BlueShiftInAppNotificationManager.m

```diff
@@ -59,7 +57,7 @@ - (void) OnApplicationEnteringBackground:(NSNotification *)notification {
 - (void) OnApplicationEnteringForeground:(NSNotification *)notification {
     /* start the timer once app enters foreground */
     
-    if ([[[BlueShift sharedInstance] config] inAppManualTriggerEnabled] == NO) {
+    if ([[[BlueShift sharedInstance] config] inAppManualTriggerEnabled] == NO && [[[BlueShift sharedInstance] config] enableInAppNotification] == YES) {
```

Comment:
what is this change?

Reply:
I have added a new check here, when app enters into the foreground, it will check for the enableInAppNotification value to start the InApp timer.